File: app/src/main/java/com/pitman2e/mutespeaker/Prefs.java
package com.pitman2e.mutespeaker;

import android.content.Context;
import android.content.SharedPreferences;

import androidx.preference.PreferenceManager;

public class Prefs {
    public static boolean getIsEnableNotification(Context context) {
        return PreferenceManager.getDefaultSharedPreferences(context).getBoolean(context.getString(R.string.PREFERENCES_ID_ENABLE_MUTE_SERVICE), true);
    }

    public static void setIsEnableNotification(Context context, boolean isEnable) {
        SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(context);
        SharedPreferences.Editor editor = preferences.edit();
        editor.putBoolean(context.getString(R.string.PREFERENCES_ID_ENABLE_MUTE_SERVICE), isEnable);
        editor.apply();
    }
}

File: app/src/main/java/com/github/pitman2e/mutespeaker/Prefs.kt
package com.github.pitman2e.mutespeaker

import android.content.Context
import androidx.preference.PreferenceManager

object Prefs {
    fun getIsEnableNotification(context: Context): Boolean {
        return PreferenceManager.getDefaultSharedPreferences(context).getBoolean(
            context.getString(R.string.PREFERENCES_ID_ENABLE_MUTE_SERVICE), true
        )
    }

    fun setIsEnableNotification(context: Context, isEnable: Boolean) {
        val preferences = PreferenceManager.getDefaultSharedPreferences(context)
        val editor = preferences.edit()
        editor.putBoolean(context.getString(R.string.PREFERENCES_ID_ENABLE_MUTE_SERVICE), isEnable)
        editor.apply()
    }
}